src/simtools/applications/derive_photon_electron_spectrum.py

```diff
@@ -69,7 +69,7 @@ def _parse(label):
     )
     config.parser.add_argument(
         "--max_amplitude",
-        help="Maximum amplitude in amplitude spectrum",
+        help="Maximum amplitude in pe for amplitude spectrum",
```

Comment:
probably better "for single p.e. spectrum"?

Reply:
Fixed.